src/cc/frontends/p4/compiler/ebpfAction.py

```diff
@@ -0,0 +1,327 @@
+from p4_hlir.hlir import p4_action, p4_field, p4_signature_ref, p4_header_instance
```

Comment:
I would request adding a form of the open source license (2 line or 2 paragraph version) to the top of each file ((c) Barefoot not PLUMgrid of course)